internal/spdxlicense/license.go

```diff
@@ -4,9 +4,25 @@ import (
 	"strings"
 )
 
+var debianLicenseLabels = map[string]string{
```

Comment:
where was this list derived from? Are we certain that `gpl-2` means exactly 2.0 and not 2.1 (or more generically the latest v2 option)?

Reply:
Good point - I'll dig a bit more into why license are formatted as `GPL-X` on the ubuntu image and what that means as far as how it fits into latest license options.

Reply:
Here are the docs for the debian [spec](https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/#license-short-name).

"For SPDX compatibility, versions with trailing dot-zeroes are considered to be equivalent to versions without (e.g., “2.0.0” is considered equal to “2.0” and “2”)."

Based on that, we can be confident that `gpl-2` means exactly 2.0.

The 'latest' is covered where a `+` is added as a suffix to the short name.

The list was derived from checking generated list at `license_list.go`.

There are potentially more to add based on the confusion surrounding `short name` equivalence in the debian space.

Cases like `2` are not covered by our generated `license_list` from [here](https://spdx.org/licenses).

Reply:
It looks like there have been specific cases added for GPL and LGPL, is there a way we could process the license list to get this for all other license types with versions in them?